synapse/lib/parser.py

```diff
@@ -644,6 +645,7 @@ def massage_vartokn(astinfo, x):
     'exprnot': s_ast.UnaryExprNode,
     'exprunary': s_ast.UnaryExprNode,
     'exprcmp': s_ast.ExprNode,
+    'exprmodulo': s_ast.ExprNode,
```

Comment:
don't need this, it is part of the 'exprproduct' rule in the grammar

Reply:
Fixed. I think I have a better understanding of how this works now.